graph/impl/src/main/java/org/jboss/windup/graph/GraphContextImpl.java

```diff
@@ -283,9 +283,15 @@ public Path getGraphDirectory()
         return graphDir;
     }
 
+    /**
+     * Returns unmodifiable Map of options.
+     * The values are created through WindupConfiguration.setOptionValue().
+     */
     @Override
     public Map<String, Object> getOptionMap()
```

Comment:
This javadoc should go on the declaring interface/base class, not on the implementation.